src/writer/ai.py

```diff
@@ -70,13 +71,20 @@ def __init__(self, token: Optional[str] = None):
 
 
         :param token: Optional; the default token for API authentication used if WRITER_API_KEY environment variable is not set up.
+        :raises RuntimeError: If an API key was not provided to initialize SDK client properly.
         """
-
-        self.client = Writer(
-            # This is the default and can be omitted
-            api_key=token,
-        )
-
+        try:
+            self.client = Writer(
+                # This is the default and can be omitted
+                api_key=token,
+            )
+        except WriterError:
```

Comment:
I don't see that we're actually checking the key, what happens if there's no internet connectivity, wouldn't this fail and tell you you don't have a key?